src/config/routes.ts

```diff
@@ -17,7 +17,7 @@ export const AppRoutes = {
     modules: '/settings/modules',
     index: '/settings',
     appearance: '/settings/appearance',
-    safeAppsPermissions: '/settings/safe-apps-permissions',
+    safeApps: '/settings/safe-apps',
```

Comment:
Not sure why `settings/safe-apps/` is better than `settings/safe-apps-permissions`, perhaps `settings/safe-apps/permissions` if you don't like the current route?

Reply:
Or you think about if we add more settings in the future we can create subsections there?

Reply:
Exactly that. For the time being, I think `/settings/safe-apps` is sufficient (until we add more).